### ISSUE TITLE
Missing Accessibility Info

### ISSUE THREAD

**Issue by kevlon300** [nightscout/AndroidAPS#732]
The "trend arrow" and all the buttons at the locale profile screen have no accessibility text. This is missing to use talkback.

**Comment by Andries-Smit** [comment]
Thank you for reporting @kevlon300 we should keep our app open for everyone.
I am working on a fix for the 2 uses your reported, please contact me to discuss your need and if you are able to validate and help to test screenreader accessibility
